tests/perf/fair_queue_perf.cc

```diff
@@ -103,10 +92,9 @@ future<> perf_fair_queue::test(bool loc) {
 
     auto invokers = local_fq.invoke_on_all([loc] (local_fq_and_class& local) {
         return parallel_for_each(boost::irange(0u, requests_to_dispatch), [&local, loc] (unsigned dummy) {
-            auto cap = local.queue(loc).tokens_capacity(double(1) / std::numeric_limits<int>::max() + double(1) / std::numeric_limits<int>::max());
-            auto req = std::make_unique<local_fq_entry>(cap, [&local, loc, cap] {
+            auto req = std::make_unique<local_fq_entry>(100, [&local, loc] {
```

Comment:
Just to double-check -- this patch replaces the calculation with hard-coded value of 100. Was the value the same before? Does it matter for the test?

Reply:
Do I understand correctly that due to returning `throttle::grab_result::grabbed` from `can_dispatch()` the capacity does not matter to the test? Because we always can dispatch (because that's how the test is now) ?

Reply:
Yes, this capacity can be any, since can_dispatch() always says "yes you can"